iRecruit-ui/src/components/Home/Home.jsx

```diff
@@ -34,7 +35,7 @@ export default function Home({
   useEffect(() => {
     const interval = setInterval(() => {
       setCarouselIndex((prevIndex) => (prevIndex + 1) % carouselContent.length);
-    }, 7800); // Change carousel content every 5 seconds
+    }, 7800);
```

Comment:
Small suggestion, whenever we have timeout values like this, its useful to show the reader how you got to this number.

For example, if we wanted 5 seconds and the call takes milliseconds we'd setup the constant like this for readability:

`const CAROUSEL_FADE_TIMEOUT = 5 * 1000; // 5 seconds`

Reply:
Oh thank you, initially wanted to use 5 seconds but the fade time and the carousel content display time was not really aligned, so I was basically doing a "line of best fit" trial and error. So I should have taken out the comment.
Also, I will take note of the constant setup.